src/sentry/incidents/logic.py

```diff
@@ -186,6 +204,16 @@ def create_incident_activity(
         kwargs={'activity_id': activity.id},
         countdown=10,
     )
+    if activity_type == IncidentActivityType.COMMENT:
+        analytics.record(
+            'incident.comment',
+            incident_id=incident.id,
+            organization_id=incident.organization_id,
+            incident_type=incident.type,
+            user_id=user.id,
+            comment=comment,
```

Comment:
Do we want to record the actual comment?

Reply:
Confirming with @saragilford

Reply:
No need for the comment, removed